docs/modules/ROOT/pages/index.adoc

```diff
@@ -875,7 +885,16 @@ Full `config/config.json` example with evm and solana relayers definitions using
       "notification_id": "notification-example",
       "signer_id": "local-signer",
       "network_type": "evm",
-      "custom_rpc_urls": ["https://your-private-rpc-url.example.com"],
+      "custom_rpc_urls": [
+        {
+          "url": "https://primary-rpc.example.com",
+          "weight": 2
```

Comment:
Given that these are docs examples and would have high visibility let's use max weight for some urls and for some let's use half of max weight for example, and even minimum weight(to document disabling).